app/modules/front-door-premium/waf-policy.tf

```diff
@@ -9,6 +9,37 @@ resource "azurerm_cdn_frontdoor_firewall_policy" "default" {
   custom_block_response_status_code = 429
 
   tags = var.common_tags
+
+  custom_rule {
+    name                           = "RateLimitHttpRequest"
+    enabled                        = true
+    priority                       = 100
+    rate_limit_duration_in_minutes = 1
+    rate_limit_threshold           = 300
+    type                           = "RateLimitRule"
+    action                         = "Block"
+
+    match_condition {
+      match_variable = "RequestMethod"
+      operator       = "Equal"
+      match_values = [
+        "GET",
+        "POST",
+        "PUT",
+        "DELETE",
+        "COPY",
+        "MOVE",
+        "HEAD",
+        "OPTIONS"
+      ]
+    }
+  }
+
+  managed_rule {
+    type    = "Microsoft_DefaultRuleSet"
+    version = "2.1"
+    action  = "Log"
+  }

```

Comment:
I don't think we should default to log only; if we want to add a default rule set then 2.1 seems sensible, but in Block mode?

This could be done separately (no service is using this module AFAIK).

Reply:
Removed - copied across from appeals back office repo assuming the whole azurerm_cdn_frontdoor_firewall_policy block would be the same, but obviously not.

No, I couldn't see anything using it, or anything bringing in the module repo for front door premium.